hardware/hardware.mk

```diff
@@ -249,6 +251,8 @@ $(BSG_MACHINE_PATH)/bsg_bladerunner_pkg.sv: $(BSG_MACHINE_PATH)/bsg_bladerunner_
 	@echo >> $@
 	@echo "parameter int bsg_machine_pod_tiles_y_gp = $(BSG_MACHINE_POD_TILES_Y);" >> $@
 	@echo "parameter int bsg_machine_pod_tiles_x_gp = $(BSG_MACHINE_POD_TILES_X);" >> $@
+	@echo "\`define BSG_MACHINE_GLOBAL_Y $(BSG_MACHINE_POD_TILES_Y)" >> $@
+	@echo "\`define BSG_MACHINE_GLOBAL_X $(BSG_MACHINE_POD_TILES_X)" >> $@
```

Comment:
@tommydcjung Would it be okay to remove these two defines? Seems like they are repetitive, we already defined them in vcs run here: https://github.com/bespoke-silicon-group/bsg_replicant/blob/main/libraries/platforms/bigblade-vcs/link.mk#L52-L53

I tried removing these two lines and the memcpy test is running fine, `BSG_MACHINE_GLOBAL_Y` and `BSG_MACHINE_GLOBAL_X` are still defined in vcs simulation. Is there anything that I didn't notice?

Reply:
I think it's used for some bind modules that needs the parameter BSG_MACHINE_GLOBAL_Y/X, but the parent module does not have access to. If you think it's repetitive, you can remove it.